bootstrap/namespaces/helmfile/helmfile.yaml

```diff
@@ -6,14 +6,14 @@ helmDefaults:
 environments:
   workload_cluster:
     values:
-      - "{{ requiredEnv "CK8S_CONFIG_PATH" }}/wc-config.yaml"
+      - "/github/workspace/{{ requiredEnv "CK8S_CONFIG_PATH" }}/wc-config.yaml"
```

Comment:
This will break local runs since this path is (probably) invalid. Please edit CK8S_CONFIG_PATH instead

Reply:
I was afraid that that would break things :) Will fix.